publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/dynamodbstream/ImportCandidateDynamoDbStreamToEventBridgeHandler.java

```diff
@@ -0,0 +1,134 @@
+import nva.commons.core.exceptions.ExceptionUtils;
+import nva.commons.core.paths.UnixPath;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.eventbridge.EventBridgeClient;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequest;
+import software.amazon.awssdk.services.eventbridge.model.PutEventsRequestEntry;
+import software.amazon.awssdk.services.s3.S3Client;
+
+public class ImportCandidateDynamoDbStreamToEventBridgeHandler
+    implements RequestHandler<DynamodbEvent, EventReference> {
+
+    public static final ImportCandidate NO_VALUE = null;
+    private static final String PROCESSING_EVENT_MESSAGE = "Processing event for identifier: {}";
+    private static final String DETAIL_TYPE_NOT_IMPORTANT = "See event topic";
+    private static final String EMITTED_EVENT_MESSAGE = "Emitted Event:{}";
+    private static final String DYNAMO_DB_STREAM_SOURCE = "DynamoDbStream";
+    private static final Logger logger = LoggerFactory.getLogger(
+        ImportCandidateDynamoDbStreamToEventBridgeHandler.class);
+    private final S3Driver s3Driver;
+    private final EventBridgeClient eventBridgeClient;
+
+    @JacocoGenerated
+    public ImportCandidateDynamoDbStreamToEventBridgeHandler() {
+        this(S3Driver.defaultS3Client().build(), defaultEventBridgeClient());
+    }
+
+    protected ImportCandidateDynamoDbStreamToEventBridgeHandler(S3Client s3Client,
+                                                                EventBridgeClient eventBridgeClient) {
+        this.s3Driver = new S3Driver(s3Client, EVENTS_BUCKET);
+        this.eventBridgeClient = eventBridgeClient;
+    }
+
+    @Override
+    public EventReference handleRequest(DynamodbEvent inputEvent, Context context) {
+        var dynamodbStreamRecord = inputEvent.getRecords().getFirst();
+        var dataEntryUpdateEvent = convertToUpdateEvent(dynamodbStreamRecord);
+        return sendEvent(dataEntryUpdateEvent, context);
+    }
+
+    private static SortableIdentifier getIdentifier(ImportCandidateDataEntryUpdate blobObject) {
+        return Optional.ofNullable(blobObject.getOldData())
+                   .map(ImportCandidate::getIdentifier)
+                   .orElseGet(() -> blobObject.getNewData().getIdentifier());
+    }
+
+    private static String toEvenBridgeDetail(EventReference eventReference) throws JsonProcessingException {
+        var detail = AwsEventBridgeDetail.newBuilder().withResponsePayload(eventReference).build();
+        return JsonUtils.dtoObjectMapper.writeValueAsString(detail);
+    }
+
+    private EventReference sendEvent(ImportCandidateDataEntryUpdate blob, Context context) {
+        logger.info(PROCESSING_EVENT_MESSAGE, getIdentifier(blob));
+        return attempt(() -> saveBlobToS3(blob)).map(blobUri -> new EventReference(blob.getTopic(), blobUri))
+                   .map(eventReference -> sendEvent(eventReference, context))
+                   .orElseThrow();
```

Comment:
What is potentially thrown there? Just the `JsonProcessingException`? See below. If it is more than this, log it properly and handle the exception in a more transparent way.